client/src/app/components/questions/Emails.tsx

```diff
@@ -2,30 +2,40 @@
 
 import { useState } from "react";
 import {BsTrash} from "react-icons/bs";
+import {sendEmail} from "../../apiServices";
```

Comment:
Instead of importing this, you should import **shareSurvey** from /services/survey.service.ts